earth2studio/io/zarr.py

```diff
@@ -32,23 +32,32 @@ class ZarrBackend:
     ----------
     file_name : str, optional
         Optional name to provide to the zarr store, if provided then this function
-        will create a directory store with this file name. If not, will create a memory store.
+        will create a directory store with this file name. If none, will create a
+        in-memory store., by default None
     chunks : dict[str, int], optional
-        An ordered dict of chunks to use with the data passed through data/coords.
+        An ordered dict of chunks to use with the data passed through data/coords, by
+        default {}
+    zarr_kwargs : dict[str, Any], optional
+        Key word arguments for zarr.Group root object, by default {"overwrite": False}
+
+    Note
+    ----
+    For keyword argument options see: https://zarr.readthedocs.io/en/stable/api/hierarchy.html
     """
 
     def __init__(
         self,
         file_name: str = None,
         chunks: dict[str, int] = {},
+        zarr_kwargs: dict[str, Any] = {"overwrite": False},
```

Comment:
Similar, can we set this to be `backend_kwargs`.

Reply:
Updated as well!